Examples to test array error tracking

domainexpert opened the pull request:
This PR contains simple examples to test that fp-analysis/klee#34 is working.

Himeshi commented on basic/array_test1.c:
```diff
@@ -0,0 +1,25 @@
+int main() {
+
+  klee_make_symbolic(arr, sizeof arr, "__arr8__arr");
+  klee_track_error(arr, "__arr8__arr_error");
```
@domainexpert Perhaps the input arrays should also be prefixed with \__arr8__? For this example it is not important for the final result, as the input is not involved in the final error expression, but it might be good to document the use of it. (Sorry that my example test case did not contain this change)

domainexpert commented:
@Himeshi Thank you so much for your comments. Somehow GitHub lost your comments after I rebased, but this is a copy from Slack:
```
@domainexpert Perhaps the input arrays should also be prefixed with  __arr8__? For this example it is not important for the final result, as the input is not involved in the final error expression, but it might be good to document the use of it. (Sorry that my example test case did not contain this change)
```
I think it may be better to leave the actual array name as `arr`, since the output error expression has `__arrXX__` removed. For example, following is the output error expression of `array_test1.c`:
```
Line 20 of /home/dcsandr/software/fp-examples/basic/array_test1.c (main): 
Output Error: (((arr_error[1] * arr[1]) + (arr_error[2] * arr[2])) / (arr[1] + arr[2]))
Absolute Bound: 1.300000e+00
```
In this way, it is easier to relate the variables of the output to the variables of the program. 

To address your concern for documentation, I agree there is a need to document this feature better in the examples. Instead of prefixing the variable names with `__arrXX__`, I added a more elaborate comments in the four examples. Please let me know your thoughts.

I also did add some modifications to correct size arguments to `klee_make_symbolic()`.
